src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.*;
```

Comment:
Following the Java coding standard, it is better to list the specific import instead of using java.util.*. : )

Reply:
Well spotted. Remember to import only specific classes as this is a coding standard violation. Remember to set IntelliJ setting so that it does not automatically merge your import statements into *.

Reply:
Noted thanks! I forgot to set this in intelliJ, will do so!